packages/edit-site/src/components/sidebar-navigation-screen-page/page-details.js

```diff
@@ -112,10 +117,10 @@ export default function PageDetails( { id } ) {
 					? postContext?.templateSlug
 					: null;
 			const _templateTitle =
-				defaultTemplateTypes && templateSlug
-					? defaultTemplateTypes.find(
+				templates && templateSlug
```

Comment:
@ramonjd is there something I might be missing here regarding your above `home` comment? Could there be a case that we also need to filter the default templates too?

Reply:
I can't remember which comment you're referring to, but I think it was just me totally overlooking that I was using `defaultTemplateTypes`.

Thanks for correcting this.